make-options.json

```diff
@@ -55,7 +55,7 @@
         "DotNetCoreInstallerV0",
         "DownloadBuildArtifactsV0",
         "DownloadFileshareArtifactsV0",
-        "DownloadGitHubReleasesV0",
+        "DownloadGitHubReleaseV0",
         "DownloadPackageV0",
```

Comment:
After completing this PR, please create corresponding PR in AzureDevOps repo.

Reply:
sure.